home_joints:
  right: {joint1: 28.61,joint2: -29.54,joint3: -6.17,joint4: -72.36,joint5: -29.92,joint6: 48.26,joint7: -8.86}
  left: {joint1: -23.09,joint2: -29.53,joint3: 4.92,joint4: -72.56,joint5: 25.49,joint6: 46.90,joint7: 9.24}